From 44fcdc1a652698e081a3dd5c988bf89142ecfd45 Mon Sep 17 00:00:00 2001
From: Christine Gharzuzi <cgharzuzi@habana.ai>
Date: Mon, 10 Jul 2023 12:01:05 +0300
Subject: [PATCH] [SW-151144] gaudi3: update freeze images

The existing images are not the correct images
that were used in freeze, this commit updates the images
to correct build that was built with the tag
'gaudi3_secureboot_rom_tapeout_final_drop_3'

Change-Id: I1c94ea918903bdb9967646294cdab9158ff09a3c
Signed-off-by: Christine Gharzuzi <cgharzuzi@habana.ai>
---

diff --git a/bin/gaudi3_arc/asic/arcboot_rom.bin b/bin/gaudi3_arc/asic/arcboot_rom.bin
index b5f8af4..eba1646 100644
--- a/bin/gaudi3_arc/asic/arcboot_rom.bin
+++ b/bin/gaudi3_arc/asic/arcboot_rom.bin
Binary files differ
diff --git a/bin/gaudi3_arc/asic/arcboot_rom.elf b/bin/gaudi3_arc/asic/arcboot_rom.elf
index 090526e..e6b20ec 100644
--- a/bin/gaudi3_arc/asic/arcboot_rom.elf
+++ b/bin/gaudi3_arc/asic/arcboot_rom.elf
Binary files differ
diff --git a/bin/gaudi3_arc/onix/arcboot_rom.bin b/bin/gaudi3_arc/onix/arcboot_rom.bin
index 85dc690..c5ffe19 100644
--- a/bin/gaudi3_arc/onix/arcboot_rom.bin
+++ b/bin/gaudi3_arc/onix/arcboot_rom.bin
Binary files differ
diff --git a/bin/gaudi3_arc/onix/arcboot_rom.elf b/bin/gaudi3_arc/onix/arcboot_rom.elf
index 4994959..21aa27e 100644
--- a/bin/gaudi3_arc/onix/arcboot_rom.elf
+++ b/bin/gaudi3_arc/onix/arcboot_rom.elf
Binary files differ
diff --git a/bin/gaudi3_arc/pldm/arcboot_rom.bin b/bin/gaudi3_arc/pldm/arcboot_rom.bin
index b5f8af4..eba1646 100644
--- a/bin/gaudi3_arc/pldm/arcboot_rom.bin
+++ b/bin/gaudi3_arc/pldm/arcboot_rom.bin
Binary files differ
diff --git a/bin/gaudi3_arc/pldm/arcboot_rom.elf b/bin/gaudi3_arc/pldm/arcboot_rom.elf
index b453cc5..2162103 100644
--- a/bin/gaudi3_arc/pldm/arcboot_rom.elf
+++ b/bin/gaudi3_arc/pldm/arcboot_rom.elf
Binary files differ
